src/list-data/request-action-mapping.ts

```diff
@@ -50,12 +50,12 @@ export const BcMapping: RequestActionMappingI = {
   // every entity type except Parishes and Private Act
   CHG: EntityTypesBC.filter(ent => ent !== EntityTypes.PAR && ent !== EntityTypes.PA),
   MVE: [
-    EntityTypes.CR,
-    EntityTypes.CC,
+    EntityTypes.CR, // will become CorpTypeCd.CONTINUE_IN
```

Comment:
What do you mean by the comments here Sev?

Reply:
Ah I see, I forgot about that.

I totally get you. It's just the name request code is a bit funky. I guess it's OK to keep as is. It makes sense now to me but the next person looking at this will probably need to do some digging to understand what's meant by that 😆 Resolving.

Reply:
Well, that's not ideal that a comment raises more questions than no comment. Any ideas how I can make this clearer?

Reply:
Maybe add a comment on a separate line before MVE: When Continuation In is selected, the following mappings to LEAR are done (something along these lines). Thoughts?